HelpSource/Guides/UsingQuarks.schelp

```diff
@@ -177,22 +177,18 @@ Hard drives die, backups fail, people make mistakes. If you keep a copy on bitbu
 
 section:: Publishing a Quark
 
-Make a git repository and push it to github, bitbucket or any publicly accessible git host.
+Make a git repository and push it to github, gitlab, bitbucket or any publicly accessible git host.
 
-tag your releases. The name of the tag is the name of the release version. Using semver (eg "1.0.1") is recommended. This provides the Quarks system with a way to list versions in the interface and to checkout a version and for people to pin dependencies to a stable release. it also provides a downloadable version on your github releases page.
+Be sure to strong::tag your releases::. The name of a tag should be the release version to which it corresponds. Using semantic versioning (eg "v1.0.1") is strongly recommended. This provides the Quarks system with the ability to list available versions and allow the user to choose which version to checkout. Version tags also allow other quark developpers to pin their dependencies to specific release versions.
```

Comment:
could you please add a link to https://semver.org/?

Reply:
will do!